src/lib/dto/request-search.dto.ts

```diff
@@ -5,7 +5,7 @@ import { QueryFilter, operators } from '../interface/query-operation.interface';
 
 export class RequestSearchDto<T> {
     @ApiPropertyOptional({ description: 'select fields', isArray: true, type: String })
-    select?: Array<keyof Partial<T>>;
+    readonly select?: Array<keyof Partial<T>>;
```

Comment:
Don't pack too many purposes into one MR

Reply:
I will separate it and request them again!
Thank you for review.